quinn-proto/src/connection.rs

```diff
@@ -118,12 +118,22 @@ pub struct Connection {
     bytes_in_flight: u64,
     /// Maximum number of bytes in flight that may be sent.
     congestion_window: u64,
-    /// The largest packet number sent when QUIC detects a loss. When a larger packet is
-    /// acknowledged, QUIC exits recovery.
-    end_of_recovery: u64,
+    /// The time when QUIC first detects a loss, causing it to enter recovery. When a packet sent
+    /// after this time is acknowledged, QUIC exits recovery.
+    recovery_start_time: u64,
     /// Slow start threshold in bytes. When the congestion window is below ssthresh, the mode is
     /// slow start and the window grows by the number of bytes acknowledged.
     ssthresh: u64,
+    /// Explicit congestion notification (ECN) counters.
```

Comment:
Nit: this hunk is inconsistent about ending doc comments with a period. My preference is not to do so consistently, since these are often sentence fragments rather than full sentences (so the elaboration part of the comment -- after the first line + newline -- would still use periods).